druid-shell/src/window.rs

```diff
@@ -165,6 +168,7 @@ pub enum MouseButton {
 //both Windows and macOS. We may want to provide polyfills for various additional cursors,
 //and we will also want to add some mechanism for adding custom cursors.
 /// Mouse cursors.
+#[derive(Clone)]
```

Comment:
`Copy`?

Reply:
I was reluctant in case it expands to hold custom cursors.